eth/eventhandler/handlers.go

```diff
@@ -273,9 +273,20 @@ func (eh *EventHandler) validatorAddedEventToShare(
 	committee := make([]*spectypes.Operator, 0)
 	for i := range event.OperatorIds {
 		operatorID := event.OperatorIds[i]
+		od, found, err := eh.nodeStorage.GetOperatorData(nil, operatorID)
```

Comment:
@olegshmuelov generally I would prefer we use the tx we're using above here. (`onShareCreation` has it) but I guess its not too bad since events are processed one by one and this is only a read op.

Reply:
added txn